scrabble.js

```diff
@@ -1,14 +1,148 @@
+const letters = {
+  A: 1,
+  B: 3,
+  C: 3,
+  D: 2,
+  E: 1,
+  F: 4,
+  G: 2,
+  H: 4,
+  I: 1,
+  J: 8,
+  K: 5,
+  L: 1,
+  M: 3,
+  N: 1,
+  O: 1,
+  P: 3,
+  Q: 10,
+  R: 1,
+  S: 1,
+  T: 1,
+  U: 1,
+  V: 4,
+  W: 4,
+  X: 8,
+  Y: 4,
+  Z: 10
+}
 
 const Scrabble = {
   score(word) {
+    let valid = /^[a-zA-Z]+$/;
 
+    if (word === "" || word.length > 7 || !word.match(valid)) {
+      throw "You must play a real word that is 7 characters or less";
+    }
+
+    let word_split = word.toUpperCase().split('');
```

Comment:
Watch out for using old ruby naming conventions. These should be `wordSplit` and `wordValue` instead of using the underscores.